tsdoc-metadata.json

```diff
@@ -0,0 +1,11 @@
+// This file is read by tools that parse documentation comments conforming to the TSDoc standard.
+// It should be published with your NPM package.  It should not be tracked by Git.

```

Comment:
Should this file be removed?

Reply:
yes, thanks, done and added to git-ignore. this won't be in our actual package for now until/unless we run api extractor as part of the release but that's fine for now